wasted-app/RestaurantListControl.cs

```diff
@@ -25,6 +27,37 @@ public static RestaurantListControl Instance
         public RestaurantListControl()
         {
             InitializeComponent();
+
+            AddAllRestaurants();
+        }
+
+        public void AddAllRestaurants()
+        {
+            ServicesController controller = new ServicesController();
+            var restaurants = controller.RestaurantService.GetAllRestaurants();
+
+            foreach (var r in restaurants)
```

Comment:
maybe not `var r` but `Restaurant restaurant`?

Reply:
I think it's fine to use it in this situation, no need to be explicit here.